tools/adios2pio-nm/adios2pio-nm.cxx

```diff
@@ -13,7 +13,7 @@ static void init_user_options(spio_tool_utils::ArgParser &ap)
       .add_opt("idir", "Directory containing data output from PIO (in ADIOS format)")
       .add_opt("nc-file", "output file name after conversion")
       .add_opt("pio-format", "output PIO_IO_TYPE. Supported parameters: \"pnetcdf\",  \"netcdf\",  \"netcdf4c\",  \"netcdf4p\"")
-      .add_opt("reduce-memory-usage", "Reduce memory usage (execution time will likely increase)")
+      .add_opt("rearr", "PIO rearranger. Supported parameters: \"subset\", \"box\". Default \"subset\".")
```

Comment:
Is the "reduce memory" option no longer required?

Reply:
I think so. We can confirm it with @tkurc ,though.

Reply:
No need for reduce memory option.